examples/with-react-helmet/.gitignore

```diff
@@ -0,0 +1,2 @@
+node_modules/
```

Comment:
Because the parent .gitignore already takes care of ignoring node_modules. As for ignoring the .DS_Store, you could put it in your own `~/.gitignore_global` as it's specific to your system (after having removed the files from this PR).